src/dialog/src/Dialog.js

```diff
@@ -257,7 +257,12 @@ class Dialog extends React.Component {
                   <Heading is="h4" size={600} flex="1">
                     {title}
                   </Heading>
-                  <IconButton appearance="ghost" icon="close" onClick={close} />
+                  <IconButton
+                    tabindex={3}
```

Comment:
We probably don't want to use a fixed tab order because then the Dialog buttons will always get focus before the Dialog content. Instead we should let the DOM order decide the tab order.